pkg/upgradeservice/handlers/config.go

```diff
@@ -379,13 +378,6 @@ func (h *Handler) SaveConfig(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	if err := upgradepreflight.Run(params); err != nil {
-		response.Error = "failed to run preflights"
-		logger.Error(errors.Wrap(err, response.Error))
-		JSON(w, http.StatusInternalServerError, response)
-		return
-	}
-
```

Comment:
what happens when deploying the app via the CLI? Or do we just not allow that yet with the new workflow

Reply:
Upgrade workflow is only supported via the UI currently.